ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -55,8 +56,38 @@
                 this.options.queryModel.set('type', 'text');
                 user.getQuerySettings().set('type', 'text');
                 break;
+            case 'newResult':
+                oldType = this.options.queryModel.get('type');
+                this.options.queryModel.set({
```

Comment:
Do we want to set the queryModel to all these empty values in this fashion?

Wondering if we just want to import ` const SearchForm = require('./search-form');` at the top and 
because everything is reset in this `newResult` case, might want to change to `this.options.queryModel.set(new SearchForm({type: 'newResult'}));` ?